src/i18n/fi/translation.json

```diff
@@ -2,18 +2,20 @@
   "Welcome to React": "Tervetuloa Reactin ja react-i18nextin pariin.",
   "slugs": {
     "about-us": "tietoa-meista",
-    "terms-of-service": "kayttoehdot",
     "accessibility-statement": "saavutettavuusseloste",
+    "cookies": "evasteet",
     "privacy-policy": "tietosuojaseloste",
-    "cookies": "evasteet"
+    "terms-of-service": "kayttoehdot",
+    "user": "kayttaja"
```

Comment:
Wouldn't it be possible to use nordic characters in the path? Or is there some problem later on the tooling if using them? `ä` `ö` `å`